extensions/hawkbit-extension-artifact-repository-mongo/src/main/java/org/eclipse/hawkbit/artifact/repository/MongoDBArtifactStore.java

```diff
@@ -216,15 +216,15 @@ private static String computeSHA1Hash(final InputStream stream, final OutputStre
     }
 
     /**
-     * Maps a list of {@link GridFSDBFile} to paged list of {@link DbArtifact}s.
+     * Maps a list of {@link GridFSDBFile} to paged list of {@link AbstractDbArtifact}s.
```

Comment:
sorry isn't your change but just found the `paged` word in the sentence is incorrect